libs/api/domains/cms/src/lib/models/frontpageSlider.model.ts

```diff
@@ -35,4 +39,9 @@ export const mapFrontpageSlider = ({
   animationJson: fields.animationJson
     ? JSON.stringify(fields.animationJson)
     : '',
```

Comment:
Consider removing this since it should no longer be needed

Reply:
Good catch, thank you